### PR TITLE
add figures for shared time uncertainty

### PR DESCRIPTION
**Pull request by qinqian**
…arison across dynamic and static dataset

### CODE REVIEW

**Comment by qinqian** [comment]
do not merge now, I'll add another angle figure later.

File: reproducibility/figures/README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ mamba env create -n pv2 -f conda/environment-gpu-upd.yml
 conda activate pv2
 pip install xdoctest or mamba install -y xdoctest
 pip install matplotlib_venn
+pip install statannotations
```
Comment (cameronraysmith) [inline reproducibility/figures/README.md:12]: we should integrate this into the dev env if it is essential

File: reproducibility/figures/dvc.lock [hunk 2]
```diff
@@ -734,8 +734,8 @@ stages:
       md5: 72a05b2966bee39e85c17b16c569733c
```
Comment (cameronraysmith) [inline reproducibility/figures/dvc.lock:734]: checked pulling from this lockfile and it works as desired

**Review by cameronraysmith — APPROVED** [review]
thanks @qinqian !

**Pull request merged**